src/core/task7.js

```diff
@@ -1,4 +1,4 @@
-export function compare(first, second) {
```

Comment:
What will be if we don't pass necessary args in pow function or the second arg will be < 1? Also, please rewrite it using simple math operator :)